Lib/test/_test_multiprocessing.py

```diff
@@ -1332,6 +1332,19 @@ def _on_queue_feeder_error(e, obj):
         self.assertTrue(not_serializable_obj.reduce_was_called)
         self.assertTrue(not_serializable_obj.on_queue_feeder_error_was_called)
 
+    def test_closed_queue_empty_exceptions(self):
```

Comment:
Since this isn't exactly obvious, I think you should have a comment here that you're testing that no error is thrown if you've never written to the Queue.